.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Should I file this?**
+Use this template to report *observed bugs in current Plotman*.  Do not use this to request tech support or troubleshooting; we are unable to provide that support here, so please use the user channel(s) on Keybase, or GitHub Discussions instead.  Do not use this to report issues with the core Chia plotter.  If you are using `main` branch, try pulling `development` to see if your problem is already fixed.
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior, e.g.:
+1. Set up config with '...'
+2. Run other programs '....'
+3. Run Plotman with '....'
+4. See error
+
```

Comment:
Nesting the triple back quotes doesn't work so just view source on this message to see the suggested source change.

<details> <summary>full configuration</summary>

```yaml
# paste your complete configuration file contents here.
```

</details>

Reply:
Couldn't quite view source but I tried to do what you're asking.

Reply:
Sorry, there's no 'view source' for the comment but there is 'edit'.

![image](https://user-images.githubusercontent.com/543719/117988025-04b54c00-b309-11eb-9be1-615e03d61ca7.png)